### PR TITLE
Feature/podaac 4852

### PR DESCRIPTION
**Pull request by sliu008**
- Create Test notebook for HOSS harmony

### CODE REVIEW

File: notebook_test.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import papermill as pm
+import argparse
+
+from os import path
+
+from utils import FileHandler
+from utils.enums import Venue
+
```
Comment (jamesfwood) [inline notebook_test.py]: Can you rename the test_notebooks folder to just **_notebooks_**?

File: notebook_test.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+def parse_args():
+    """
+    Parses the program arguments
+    Returns
+    -------
+    args
+    """
```
Comment (jamesfwood) [inline notebook_test.py]: Can you rename the notebook_test.py to something a little more descriptive?  Maybe like **_notebook_collection_test.py_**?  

Maybe rename the hoss_subsetter_test.ipynb to something more descriptive too?  Like **_hoss_collection_test.ipynb_**? 
Thanks!

**Comment by jamesfwood** [comment]
Can you add a README.md with how to run it?  Can you add an example how to run the notebook with an example input collection that works?

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [unreleased]
 
 ### Added
+- **PODAAC-4852**
+  - Add HOSS jupyter test notebook
 ### Deprecated
 ### Removed
 ### Fixed
```
Comment (jamesfwood) [inline CHANGELOG.md:15]: Can you add a README.md with how to run it? Can you add an example how to run the notebook with an example input collection that works?

Comment (sliu008) [reply CHANGELOG.md:15]: Updated readme, the notebook has a default podaac collection in ops that the notebook runs with.

**Review by jamesfwood — APPROVED** [review]
Looks good.  Thanks!

**Pull request merged**